redis/Module.md

```diff
@@ -0,0 +1,44 @@
+## Overview
+
+The Ballerina connector for Redis allows you to connect to Redis and manipulate key-value data stored in a Redis database. You can use the connector to manipulate string, hash, list, set, and sorted set data types. The functionality of the Ballerina connector for Redis is limited to the use of Redis as a database and cache. The message broker functionality of Redis is not yet exposed through this module.
+
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+* Make sure Redis cache is running in the local environment or use Redis cache running on the cloud (for example, [Redis Enterprise Cloud](https://redislabs.com/redis-enterprise-cloud/overview/))
```

Comment:
There are two closing brackets. Remove one and insert a full stop at the end of the sentence.

Reply:
There should be two closing brackets